vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -929,22 +929,48 @@ void saveToJson(JsonWriter* writer, const WALLET_PROPERTIES_ST& data) {
   }
   writer->EndObject();
 
-  writer->String("parameters_choices");
+  writer->String("parameters");
+  writer->StartObject();
+  writer->String("adFree");
```

Comment:
Where does `adFree` come from as I see no reference to this anywhere?

Reply:
@tmancey check that for `adFree` https://github.com/brave/brave-core/pull/2228/files#diff-8e22b475f70e7eca00ac43fcfb993885R873